### PR TITLE
Adding paper 16a

### PR DESCRIPTION
**Pull request by ratnadeepb**
Ok this looks reasonably clean except for the .gitignore file. Let me know if that needs to be removed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+papers/.DS_Store
```
Comment (gparmer) [inline .gitignore:1]: Do you want me to just manually remove this after the pull? Is it too hard to get rid of?

**Comment by gparmer** [comment]
.DS_STORE still here. See inline comment.

File: .gitignore (hunk 1 above)
Comment (ratnadeepb) [reply .gitignore:1]: I think that's just the git ignore entry. I can have git ignore ignore itself.

Comment (gparmer) [reply .gitignore:1]: LOL, I'm an idiot. Thanks for bearing with me.

**Pull request merged**